src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -112,32 +108,32 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
         if (Globals.prefs.getBoolean(JabRefPreferences.SPECIALFIELDSENABLED)) {
             if (Globals.prefs.getBoolean(JabRefPreferences.SHOWCOLUMN_RANKING)) {
                 JMenu rankingMenu = new JMenu();
-                RightClickMenu.populateSpecialFieldMenu(rankingMenu, Rank.getInstance(), frame);
+                RightClickMenu.populateSpecialFieldMenu(rankingMenu, SpecialField.RANK, frame);
                 add(rankingMenu);
             }
 
             // TODO: multiple handling for relevance and quality-assurance
             // if multiple values are selected ("if (multiple)"), two options (set / clear) should be offered
             // if one value is selected either set or clear should be offered
             if (Globals.prefs.getBoolean(JabRefPreferences.SHOWCOLUMN_RELEVANCE)) {
-                add(Relevance.getInstance().getValues().get(0).getMenuAction(frame));
```

Comment:
There is just one value used here, so, as far as I see, no potential for a forEach.